static/app/views/releases/list/index.tsx

```diff
@@ -285,6 +286,21 @@ class ReleasesList extends DeprecatedAsyncView<Props, State> {
     });
   };
 
+  filterKeys = () => {
```

Comment:
Looks like the filter keys are constant. Could we define this as a property (`FILTER_KEYS`) outside of the component to prevent unnecessary rerenders?